README.md

```diff
@@ -212,6 +212,58 @@ non-technical people, focus on the fact that `.txt` email
 is more ***accessible*** and prevents your messages getting
 blocked by spam filters.
 
+### High volume of emails when running automated tests?
+When testing functions which will subsequently call methods in third party libraries,
+your tests no longer need to run through those methods,
+you can instead assume that the third party method will give back what they say they will give back
+(since these tests should already exist in the library you are using).
+
+Once you are sure there are [sufficient tests in place](https://codecov.io/github/dwyl/sendemail?branch=master) for the method you will stub, you may proceed with a test double.
+For this library there are significant benefits for using a test double for the `email` method (see #49).
```

Comment:
We should probably link this to the issue itself for clarity but not a blocker. We can link and expand on this with the content of the issue at a later date so the issue can be closed 👍